CMakeLists.txt

```diff
@@ -91,6 +92,7 @@ include_directories(../tvm/dlpack/include/)
 include_directories(../dlpack/include/)
 include_directories(build/include/)
 include_directories(${CMAKE_CURRENT_BINARY_DIR}/include)
+include_directories(${CMAKE_CURRENT_BINARY_DIR}/ngraph/src/ext_ngraph-build/mkl/src/ext_mkl/include)
```

Comment:
if USE_MKLDNN is defined, can we link to DO MKLDNN's mkl instead of hardcoding to ext_ngraph's mkl.

Reply:
If USE_MKLDNN is defined, we use DO's mklml and this path is empty, ngraph never builds mkl. If USE_MKLDNN is not defined, nGraph build's it's own and we use that one, because this path has files in it.